### DISCUSSION TITLE
Changing templates output directory

### DISCUSSION THREAD

**Discussion by avidseeker** [11ty/eleventy#1801] [Q&A] [answered]
Structure:
```
input: src
       > posts
            > 1-blog-post.md
            > 2-another-post.md
            > ...
       ...
output: dist
```
When serving the site, the post permalinks are `example.com/posts/1-blog-post/`
Question: how can I make the permalinks: `example.com/1-blog-post/`

Note: I transferring posts to `/src` get the required result but It's important to have `/posts` folder so that the folder does not get messy.

**Comment by pdehaan** [comment, marked as answer]
@AvidSeeker Try creating a /src/posts/posts.json file with the following:

```json
{
  "permalink": "/{{ page.fileSlug }}/"
}
```

This will set the [`permalink`](https://www.11ty.dev/docs/permalinks/) key in each template of your /src/posts/ folder to the template's [`fileSlug`](https://www.11ty.dev/docs/data-eleventy-supplied/#fileslug) and move the generated files into the root directory.
